ni_measurementlink_service/measurement/service.py

```diff
@@ -370,12 +375,20 @@ def _make_annotations_dict(
 
         return annotations
 
-    def _enum_to_annotations_value(self, enum_type: Type[Enum]) -> str:
-        if not any(member.value == 0 for member in enum_type):
-            raise ValueError("The enum does not have a value for 0.")
+    def _enum_to_annotations_value(
+        self, enum_type: Union[Type[Enum], Type[EnumTypeWrapper]]
```

Comment:
Consider defining a type alias instead of duplicating this type.

Reply:
Added `SupportedEnumType` alias.